godot-solana-sdk.gdextension

```diff
@@ -13,6 +13,7 @@ windows.template_release.x86_64="libgodot-solana-sdk.windows.template_release.x8
 windows.template_debug.x86_64="libgodot-solana-sdk.windows.template_debug.x86_64.so"
 macos.template_release="libgodot-solana-sdk.macos.template_release.framework"
 macos.template_debug="libgodot-solana-sdk.macos.template_debug.framework"
+macos.arm64="libgodot-solana-sdk.macos.template_debug.framework"
```

Comment:
Does it work with macos.template_debug.arm64="libgodot-solana-sdk.macos.template_debug.framework"?

Reply:
Then we can add `macos.template_release.arm64="libgodot-solana-sdk.macos.template_release.framework"`

Reply:
Otherwise the exported game might have debug symbols included

Reply:
I don't know why but only macos.arm64= works...
how to make difference for release and debug lib, I don't see in the code/doc...

Reply:
I see, then we keep these changes for now. It should not matter too much if debug symbols are present in exports. Again, thanks for the contribution.